<!DOCTYPE html>

<html lang="en" data-content_root="../">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Multi-omics data preprocessing &#8212; Hi-C_paper 1.0.1 documentation</title>
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=a746c00c" />
    <link rel="stylesheet" type="text/css" href="../_static/nature.css?v=0f882399" />
    <script src="../_static/documentation_options.js?v=292eb321"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Identification of intrachromosomal communities" href="4_Identification%20of%20intrachromosomal%20communities.html" />
    <link rel="prev" title="Hi-C data preprocessing" href="2_Hi-C_data%20preprocessing.html" /> 
  </head><body>
    <div class="related" role="navigation" aria-label="related navigation">
      <h3>Navigation</h3>
      <ul>
        <li class="right" style="margin-right: 10px">
          <a href="../genindex.html" title="General Index"
             accesskey="I">index</a></li>
        <li class="right" >
          <a href="4_Identification%20of%20intrachromosomal%20communities.html" title="Identification of intrachromosomal communities"
             accesskey="N">next</a> |</li>
        <li class="right" >
          <a href="2_Hi-C_data%20preprocessing.html" title="Hi-C data preprocessing"
             accesskey="P">previous</a> |</li>
        <li class="nav-item nav-item-0"><a href="../index.html">Hi-C_paper 1.0.1 documentation</a> &#187;</li>
        <li class="nav-item nav-item-this"><a href="">Multi-omics data preprocessing</a></li> 
      </ul>
    </div>  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section id="multi-omics-data-preprocessing">
<h1>Multi-omics data preprocessing<a class="headerlink" href="#multi-omics-data-preprocessing" title="Link to this heading">¶</a></h1>
<nav class="contents local" id="contents">
<ul class="simple">
<li><p><a class="reference internal" href="#preprocess-omics-map2hic" id="id3">1. preprocess_omics_map2hic</a></p>
<ul>
<li><p><a class="reference internal" href="#arguments" id="id4">Arguments</a></p></li>
<li><p><a class="reference internal" href="#data-path" id="id5">Data path</a></p></li>
</ul>
</li>
<li><p><a class="reference internal" href="#preprocess-omics-heatmap" id="id6">2. preprocess_omics_heatmap</a></p>
<ul>
<li><p><a class="reference internal" href="#id1" id="id7">Arguments</a></p></li>
<li><p><a class="reference internal" href="#id2" id="id8">Data path</a></p></li>
</ul>
</li>
</ul>
</nav>
<p>The section is designed for data preprocessing of raw omics data.</p>
<section id="preprocess-omics-map2hic">
<h2><a class="toc-backref" href="#id3" role="doc-backlink">1. preprocess_omics_map2hic</a><a class="headerlink" href="#preprocess-omics-map2hic" title="Link to this heading">¶</a></h2>
<p>For other omics data, they are first organized into bed format files.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">dna</span><span class="o">.</span><span class="n">preprocess_omics_map2hic</span><span class="p">(</span><span class="n">multi_omics</span><span class="p">)</span>
</pre></div>
</div>
<section id="arguments">
<h3><a class="toc-backref" href="#id4" role="doc-backlink">Arguments</a><a class="headerlink" href="#arguments" title="Link to this heading">¶</a></h3>
<p><strong>Required arguments:</strong></p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">multi_omics</span></code>: Only accept ‘gene expression’, ‘nucleosome density’, and ‘histone marker’ currently. Other omics data can also be added if they have been appropriately prepared.</p></li>
</ul>
</section>
<section id="data-path">
<h3><a class="toc-backref" href="#id5" role="doc-backlink">Data path</a><a class="headerlink" href="#data-path" title="Link to this heading">¶</a></h3>
<p><strong>input data path:</strong>
<cite>/in_data_folder/multi_omics</cite></p>
<p><strong>output data path:</strong>
<cite>/out_data_folder/multi_omics/resolution/out_data</cite></p>
</section>
</section>
<section id="preprocess-omics-heatmap">
<h2><a class="toc-backref" href="#id6" role="doc-backlink">2. preprocess_omics_heatmap</a><a class="headerlink" href="#preprocess-omics-heatmap" title="Link to this heading">¶</a></h2>
<p>Then, multi-omics datasets are mapped to the Hi-C adjacency matrices and heatmaps are drawn based on the genomic feature matrices.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">dna</span><span class="o">.</span><span class="n">preprocess_omics_heatmap</span><span class="p">(</span><span class="n">multi_omics</span><span class="p">,</span> <span class="n">type_of_calculation</span><span class="p">,</span> <span class="n">fig_dpi</span> <span class="o">=</span> <span class="mi">300</span><span class="p">)</span>
</pre></div>
</div>
<section id="id1">
<h3><a class="toc-backref" href="#id7" role="doc-backlink">Arguments</a><a class="headerlink" href="#id1" title="Link to this heading">¶</a></h3>
<p><strong>Required arguments:</strong></p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">multi_omics</span></code>: Only accept ‘gene expression’, ‘nucleosome density’, and ‘histone marker’ currently. Other omics data can also be added if they have been appropriately prepared.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">type_of_calculation</span></code>: Two options for the value of multi-omics data in the crossed window. It should be setted as ‘mean’ or ‘max’.</p></li>
</ul>
<p><strong>Optional arguments:</strong></p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">fig_dpi</span></code>: Figure resolution in dots per inch. The default is 300.</p></li>
</ul>
</section>
<section id="id2">
<h3><a class="toc-backref" href="#id8" role="doc-backlink">Data path</a><a class="headerlink" href="#id2" title="Link to this heading">¶</a></h3>
<p><strong>output data path:</strong>
With the output of the previous function as input, the zscore matrix and heatmap are generated to <cite>/out_data_folder/multi_omics/resolution/out_plot</cite>.</p>
</section>
</section>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
  <div>
    <h3><a href="../index.html">Table of Contents</a></h3>
    <ul>
<li><a class="reference internal" href="#">Multi-omics data preprocessing</a><ul>
<li><a class="reference internal" href="#preprocess-omics-map2hic">1. preprocess_omics_map2hic</a><ul>
<li><a class="reference internal" href="#arguments">Arguments</a></li>
<li><a class="reference internal" href="#data-path">Data path</a></li>
</ul>
</li>
<li><a class="reference internal" href="#preprocess-omics-heatmap">2. preprocess_omics_heatmap</a><ul>
<li><a class="reference internal" href="#id1">Arguments</a></li>
<li><a class="reference internal" href="#id2">Data path</a></li>
</ul>
</li>
</ul>
</li>
</ul>

  </div>
  <div>
    <h4>Previous topic</h4>
    <p class="topless"><a href="2_Hi-C_data%20preprocessing.html"
                          title="previous chapter">Hi-C data preprocessing</a></p>
  </div>
  <div>
    <h4>Next topic</h4>
    <p class="topless"><a href="4_Identification%20of%20intrachromosomal%20communities.html"
                          title="next chapter">Identification of intrachromosomal communities</a></p>
  </div>
  <div role="note" aria-label="source link">
    <h3>This Page</h3>
    <ul class="this-page-menu">
      <li><a href="../_sources/content/3_Multi-omics_data preprocessing.rst.txt"
            rel="nofollow">Show Source</a></li>
    </ul>
   </div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="../search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="related" role="navigation" aria-label="related navigation">
      <h3>Navigation</h3>
      <ul>
        <li class="right" style="margin-right: 10px">
          <a href="../genindex.html" title="General Index"
             >index</a></li>
        <li class="right" >
          <a href="4_Identification%20of%20intrachromosomal%20communities.html" title="Identification of intrachromosomal communities"
             >next</a> |</li>
        <li class="right" >
          <a href="2_Hi-C_data%20preprocessing.html" title="Hi-C data preprocessing"
             >previous</a> |</li>
        <li class="nav-item nav-item-0"><a href="../index.html">Hi-C_paper 1.0.1 documentation</a> &#187;</li>
        <li class="nav-item nav-item-this"><a href="">Multi-omics data preprocessing</a></li> 
      </ul>
    </div>
    <div class="footer" role="contentinfo">
    &#169; Copyright 2024, Gianna.Liu.
      Created using <a href="https://www.sphinx-doc.org/">Sphinx</a> 7.3.7.
    </div>
  </body>
</html>