<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Installation &mdash; DNAICI 0.1 documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=2709fde1"></script>
        <script src="_static/doctools.js?v=9a2dae69"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Hi-C data preprocessing" href="2_Hi-C_data%20preprocessing.html" />
    <link rel="prev" title="DNAICI: Differential Network Analysis in Intra-chromosomal Community Interaction" href="index.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            DNAICI
          </a>
              <div class="version">
                0.1.0
              </div>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">Installation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#download">Download</a></li>
<li class="toctree-l2"><a class="reference internal" href="#requirements">Requirements</a></li>
<li class="toctree-l2"><a class="reference internal" href="#initialize-package">Initialize package</a></li>
<li class="toctree-l2"><a class="reference internal" href="#arguments">Arguments</a></li>
<li class="toctree-l2"><a class="reference internal" href="#package-contents">Package Contents</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="2_Hi-C_data%20preprocessing.html">Hi-C data preprocessing</a></li>
<li class="toctree-l1"><a class="reference internal" href="3_Multi-omics_data%20preprocessing.html">Multi-omics data preprocessing</a></li>
<li class="toctree-l1"><a class="reference internal" href="4_Identification%20of%20intrachromosomal%20communities.html">Identification of intrachromosomal communities</a></li>
<li class="toctree-l1"><a class="reference internal" href="5_Enrichment%20of%20genomic%20features.html">Enrichment of genomic feature</a></li>
<li class="toctree-l1"><a class="reference internal" href="6_Differential%20interacting%20nodes.html">Differential interacting nodes</a></li>
<li class="toctree-l1"><a class="reference internal" href="7_Enrichment%20and%20network%20analysis.html">Enrichment and network analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="Paper%20related%20files.html">Paper Link</a></li>
<li class="toctree-l1"><a class="reference internal" href="Paper%20related%20files.html#supplementary-files">Supplementary files</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">DNAICI</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Installation</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/1_Installation.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="installation">
<h1>Installation<a class="headerlink" href="#installation" title="Link to this heading"></a></h1>
<nav class="contents local" id="contents">
<ul class="simple">
<li><p><a class="reference internal" href="#download" id="id1">Download</a></p></li>
<li><p><a class="reference internal" href="#requirements" id="id2">Requirements</a></p></li>
<li><p><a class="reference internal" href="#initialize-package" id="id3">Initialize package</a></p></li>
<li><p><a class="reference internal" href="#arguments" id="id4">Arguments</a></p></li>
<li><p><a class="reference internal" href="#package-contents" id="id5">Package Contents</a></p></li>
</ul>
</nav>
<section id="download">
<h2><a class="toc-backref" href="#id1" role="doc-backlink">Download</a><a class="headerlink" href="#download" title="Link to this heading"></a></h2>
<p>Dnaici is written in Python and should be run in Python. The package can be downloaded from the  <a class="reference external" href="https://github.com/differential-network-analysis/dnaici">DNAICI GitHub page</a>. Alternatively, you can use the following command:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>wget<span class="w"> </span>https://github.com/differential-network-analysis/dnaici.git
</pre></div>
</div>
<p>Currently, DNAICI cannot be downloaded using pip install dnaici.</p>
</section>
<section id="requirements">
<h2><a class="toc-backref" href="#id2" role="doc-backlink">Requirements</a><a class="headerlink" href="#requirements" title="Link to this heading"></a></h2>
<p>HOMER and Bedtools are required by data preprocessing. Java environment are required by clustering algorithm SLM (Smart Local Moving: ModularityOptimizer.jar).</p>
</section>
<section id="initialize-package">
<h2><a class="toc-backref" href="#id3" role="doc-backlink">Initialize package</a><a class="headerlink" href="#initialize-package" title="Link to this heading"></a></h2>
<p>Change to the correct working directory in Python first. It should be inside <strong>dnaici-master/dnaici/</strong></p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="kn">import</span> <span class="nn">os</span>
<span class="n">os</span><span class="o">.</span><span class="n">chdir</span><span class="p">(</span><span class="n">path</span><span class="o">-</span><span class="n">to</span><span class="o">-</span><span class="n">dnaici</span><span class="p">)</span>
</pre></div>
</div>
<p>Then dnaici could be imported successfully</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="kn">from</span> <span class="nn">dnaici</span> <span class="kn">import</span> <span class="n">dnaici</span>
<span class="n">dna</span> <span class="o">=</span> <span class="n">dnaici</span><span class="o">.</span><span class="n">DNAICI</span><span class="p">(</span><span class="n">in_data_folder</span><span class="p">,</span>
                    <span class="n">out_data_folder</span><span class="p">,</span>
                    <span class="n">cohort</span> <span class="o">=</span> <span class="s1">&#39;untreated&#39;</span><span class="p">,</span>
                    <span class="n">chromosome</span> <span class="o">=</span> <span class="p">[</span><span class="s1">&#39;chr18&#39;</span><span class="p">,</span> <span class="s1">&#39;chr19&#39;</span><span class="p">],</span>
                    <span class="n">resolution</span> <span class="o">=</span> <span class="mi">500000</span><span class="p">)</span>
</pre></div>
</div>
</section>
<section id="arguments">
<h2><a class="toc-backref" href="#id4" role="doc-backlink">Arguments</a><a class="headerlink" href="#arguments" title="Link to this heading"></a></h2>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">in_data_folder</span></code>: Input data directory where Hi-C, gene expression, nucleosome density, histone marker, genome index, and other necessary datasets are deposited.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">out_data_folder</span></code>: Output data directory where you hope to store processed Hi-C, gene expression, nucleosome density, histone marker and other analysis results.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">cohort</span></code>: The experimental condition for input data. The default is ‘untreated’, representing untreated MCF7 cell line. It can be changed to ‘tamr’, which represent tamoxifen-resistant MCF7 cell line for demo data, or ‘t0’ and ‘t1’ for full data. If you are using your own datasets, make sure the input is consistent with the name of your dataset, .</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">chromosome</span></code>: List of chromosomes you want to investigate. The default is [‘chr18’, ‘chr19’], designed for demo data. If you want to get the complete results for 23 chromosomes, please let the input be ‘whole_genome’.</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">resolution</span></code>: The resolution is used to divide the chromosome into equally sized window bin for analysis. The default is 500000, representing 500 kilobase pair. Due to the computational complexity, a relatively large input (e.g. 1000000 or 500000), which represent a low resolution, is recommended for the first round calculation.</p></li>
</ul>
</section>
<section id="package-contents">
<h2><a class="toc-backref" href="#id5" role="doc-backlink">Package Contents</a><a class="headerlink" href="#package-contents" title="Link to this heading"></a></h2>
<p>The package folder will contain the following:</p>
<ul class="simple">
<li><p><code class="docutils literal notranslate"><span class="pre">demo</span></code>: The input demo data for users</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">dnaici</span></code>: The main program of DANICI</p></li>
<li><p><code class="docutils literal notranslate"><span class="pre">README.md</span></code>: Short guides to DANICI and demo</p></li>
</ul>
</section>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="index.html" class="btn btn-neutral float-left" title="DNAICI: Differential Network Analysis in Intra-chromosomal Community Interaction" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="2_Hi-C_data%20preprocessing.html" class="btn btn-neutral float-right" title="Hi-C data preprocessing" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2024, Zhihao Yao &amp; Junbai Wang.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>