Search.setIndex({"alltitles": {"1. cluster_for_hic": [[3, "cluster-for-hic"]], "1. diegs_enrichment": [[6, "diegs-enrichment"]], "1. enrichment_permutation": [[4, "enrichment-permutation"]], "1. estimate_resolution": [[1, "estimate-resolution"]], "1. network_centrality": [[5, "network-centrality"]], "1. preprocess_omics_map2hic": [[2, "preprocess-omics-map2hic"]], "2. diegs_subnetwork": [[6, "diegs-subnetwork"]], "2. enrichment_heatmap": [[4, "enrichment-heatmap"]], "2. estimate_community_size": [[3, "estimate-community-size"]], "2. network_sigNodes": [[5, "network-signodes"]], "2. preprocess_hic_tag2homer": [[1, "preprocess-hic-tag2homer"]], "2. preprocess_omics_heatmap": [[2, "preprocess-omics-heatmap"]], "3. cluster_community_structure": [[3, "cluster-community-structure"]], "3. network_comparison": [[5, "network-comparison"]], "3. preprocess_hic_homer2bed": [[1, "preprocess-hic-homer2bed"]], "4. cluster_super_network": [[3, "cluster-super-network"]], "Arguments": [[0, "arguments"], [1, "arguments"], [1, "id1"], [1, "id2"], [2, "arguments"], [2, "id1"], [3, "arguments"], [3, "id1"], [3, "id2"], [3, "id4"], [4, "arguments"], [4, "id1"], [5, "arguments"], [5, "id1"], [5, "id2"], [6, "arguments"], [6, "id1"]], "Contents:": [[8, "contents"]], "DNAICI: Differential Network Analysis in Intra-chromosomal Community Interaction": [[8, "dnaici-differential-network-analysis-in-intra-chromosomal-community-interaction"]], "Data path": [[1, "data-path"], [1, "id3"], [2, "data-path"], [2, "id2"], [3, "data-path"], [3, "id3"], [3, "id5"], [4, "data-path"], [5, "data-path"], [5, "id3"], [6, "data-path"]], "Differential interacting nodes": [[5, "differential-interacting-nodes"]], "Download": [[0, "download"]], "Enrichment and network analysis": [[6, "enrichment-and-network-analysis"]], "Enrichment of genomic feature": [[4, "enrichment-of-genomic-feature"]], "Hi-C data preprocessing": [[1, "hi-c-data-preprocessing"]], "Identification of intrachromosomal communities": [[3, "identification-of-intrachromosomal-communities"]], "Install": [[0, "install"]], "Installation": [[0, "installation"]], "Multi-omics data preprocessing": [[2, "multi-omics-data-preprocessing"]], "Package Contents": [[0, "package-contents"]], "Package Installation": [[0, "package-installation"]], "Requirements": [[0, "requirements"]], "Supplementary Files": [[7, "supplementary-files"]]}, "docnames": ["1_Installation", "2_Hi-C_data preprocessing", "3_Multi-omics_data preprocessing", "4_Identification of intrachromosomal communities", "5_Enrichment of genomic features", "6_Differential interacting nodes", "7_Enrichment and network analysis", "Supplementary Files", "index"], "envversion": {"sphinx": 61, "sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2, "sphinx.ext.intersphinx": 1}, "filenames": ["1_Installation.rst", "2_Hi-C_data preprocessing.rst", "3_Multi-omics_data preprocessing.rst", "4_Identification of intrachromosomal communities.rst", "5_Enrichment of genomic features.rst", "6_Differential interacting nodes.rst", "7_Enrichment and network analysis.rst", "Supplementary Files.rst", "index.rst"], "indexentries": {}, "objects": {}, "objnames": {}, "objtypes": {}, "terms": {"": 5, "0": [1, 3, 4, 5, 6, 7], "01": 4, "02": 3, "05": [5, 6, 7], "1": 7, "10": [3, 7], "100": [3, 4, 5], "100kb": 1, "1378": 7, "15": 7, "2": 7, "20": 3, "23": [0, 3], "25": 7, "2681": 7, "3": 7, "300": [1, 2, 3, 4, 5, 6], "4": 7, "5": 7, "500": 0, "500000": 0, "500kb": 1, "50kb": [1, 7], "515": 7, "6": 7, "66": 7, "7": 7, "8": 7, "9": 7, "9c": 7, "For": [2, 5, 8], "If": [0, 1, 3], "In": [1, 6], "It": [0, 2, 3, 5], "The": [0, 1, 2, 3, 4, 5, 6, 7, 8], "Then": [1, 2, 3, 4, 5, 6], "These": 7, "With": [2, 5], "about": [5, 8], "abov": [5, 6], "absolut": 7, "accept": [2, 3], "access": 0, "accord": 7, "activ": 7, "ad": 2, "addit": 1, "adjac": [1, 2], "after": [3, 4, 5, 6], "again": 3, "algorithm": [0, 3], "all": [1, 3, 5, 6, 7], "also": 2, "altern": [0, 3], "an": 7, "analysi": [0, 5, 7], "analyz": [3, 8], "annot": 7, "anoth": 5, "appli": 1, "appropri": 2, "ar": [0, 1, 2, 3, 4, 5, 6, 7, 8], "assign": 5, "avail": 0, "averag": 1, "base": [2, 6, 7], "bed": [1, 2], "bedtool": [0, 1], "been": 2, "befor": 1, "belong": 7, "between": [1, 5, 7], "betwenn": 5, "bin": [0, 7], "biolog": 7, "blue": 7, "both": [0, 4, 5], "bp": 7, "build": 1, "c": [0, 2, 3, 5, 7, 8], "cal_typ": 1, "calcul": 1, "can": [0, 1, 2, 3, 5, 6], "cannot": 0, "cell": [0, 7], "central": 5, "chang": [0, 5, 6], "check": [1, 3], "chr1": [1, 3, 5], "chr18": [0, 8], "chr19": [0, 8], "chr2": [1, 3, 5], "chromosom": [0, 1, 3, 4, 5, 6, 7], "click": 7, "cluster": [0, 3], "cluster_to_json": 3, "cluter": 3, "cohort": [0, 1, 3, 5, 6, 8], "cohort1": [1, 5, 6], "cohort2": [1, 5, 6], "color": 7, "command": 0, "common": 5, "commun": [4, 7], "compar": [1, 4], "comparison": 1, "complet": [0, 5, 6], "compos": 7, "condit": [0, 1, 7], "consist": [3, 5], "consol": 3, "consum": [1, 3], "contain": [0, 7], "convert": 1, "cross": 2, "current": [0, 2], "cutoff": [1, 3, 4], "cutoff4proport": 3, "cytoscap": 7, "data": [0, 8], "dataset": [2, 3, 5], "david": [6, 7], "default": [0, 1, 2, 3, 4, 5, 6, 8], "deg": 7, "demo": [0, 8], "densiti": [0, 2], "deposit": [0, 5], "design": [1, 2, 8], "detail": 8, "determin": [3, 7], "dieg": [6, 7], "differ": [1, 5], "differenti": [6, 7], "differential_network_analysi": [5, 6], "directori": 0, "distanc": 7, "distribut": 7, "divid": [0, 4], "dna": [0, 1, 2, 3, 4, 5, 6], "dnaici": 0, "do": [4, 5], "dot": [1, 2, 3, 4, 5, 6], "download": 7, "draw": 4, "drawn": 2, "e": [1, 3, 8], "each": [3, 4, 5, 6, 7], "edg": 7, "enhanc": [6, 7], "enrich": [7, 8], "enrichment_permut": 5, "environ": 0, "epimapp": 0, "equal": [0, 1], "estim": [1, 3], "euclidean": 7, "evalu": 7, "experiment": [0, 1], "explor": 1, "export": [1, 3, 5, 6], "express": [0, 2, 6, 7], "featur": [2, 3, 5, 6, 7, 8], "feautr": 4, "fig_dpi": [1, 2, 3, 4, 5, 6], "figur": [1, 2, 3, 4, 5, 6, 7], "file": [1, 2, 8], "filter": 1, "find": [3, 5], "finish": 3, "first": [1, 2, 3, 7], "firstli": 5, "fit": 7, "fold": [6, 7], "foldchang": 6, "folder": 0, "follow": [0, 5, 7], "format": [1, 2, 3, 5], "found": 3, "from": [0, 4, 6, 7], "full": 0, "function": [2, 3, 7], "g": 8, "gaussian": 7, "gene": [0, 2, 6, 7], "gener": [2, 3], "genom": [1, 2, 3, 5, 6, 7, 8], "genome_vers": 1, "get": [0, 3, 6], "github": 0, "given": 5, "go": [6, 7], "grai": 7, "h3k27ac": 7, "h3k27me3": 7, "h3k4me1": 7, "h3k4me3": 7, "h3k9me3": 7, "have": [2, 5], "heatmap": [1, 2, 4], "hg19": [0, 1], "hg38": 1, "hi": [0, 2, 3, 5, 7, 8], "hic_community_data": [3, 5], "hic_community_figur": [3, 4], "hic_data": [1, 3, 4, 5], "hic_interaction_b": 1, "hic_interaction_hom": 1, "hicup": 1, "hicup_process": 1, "higher": 4, "highlight": 7, "histon": [0, 2, 6, 7], "homer": [0, 1], "i": [0, 1, 2, 3, 4, 5, 6, 7, 8], "identif": 8, "identifi": [3, 5, 6, 7], "import": 0, "in_data_fold": [0, 1, 2], "inch": [1, 2, 3, 4, 5, 6], "includ": 1, "incorpor": 7, "inform": [5, 6], "input": [0, 1, 2], "instal": 8, "instruct": 8, "interact": [1, 3, 6, 7], "intra": [1, 7], "intrachromosom": 8, "investig": [0, 1, 3, 5], "iter": 3, "its": 7, "jar": [0, 3], "java": 0, "json": 3, "kegg": 7, "kilobas": 0, "left": 7, "let": 0, "like": [3, 5], "line": [0, 7], "link": 7, "linux": 0, "list": 0, "locat": [3, 7], "louvain": 3, "lower": 4, "maco": 0, "manual": 1, "manuscript": 7, "map": 2, "marker": [0, 2, 6, 7], "matric": [1, 2, 3, 7], "matrix": 2, "max": 2, "maxim": 7, "mcf7": [0, 7], "mcf7tr": 7, "md": 8, "mean": 2, "method": [3, 6], "mf": 7, "mincluster_s": 3, "minim": [3, 7], "minimum": 3, "modul": 3, "modular": 3, "modularity_funct": 3, "modularityoptim": [0, 3], "molecular": 7, "more": [3, 8], "multi": 8, "multi_om": 2, "multilevel": 3, "n_iter": 3, "n_random_start": 3, "name": [6, 7], "need": [3, 5, 6], "network": [3, 5, 7], "network_signod": 6, "next": [1, 3, 5], "node": [3, 6, 7, 8], "now": 5, "nucleosom": [0, 2], "number": [3, 4, 5], "obtain": [5, 6], "omic": [1, 8], "one": 7, "onli": 2, "ontologi": 7, "oper": 0, "optim": 3, "optimization_algorithm": 3, "option": [1, 2, 3, 4, 5, 6], "orang": 7, "organ": [2, 3], "origin": 3, "other": [0, 2, 3, 5], "out": 7, "out_data": 2, "out_data_fold": [0, 1, 2, 3, 4, 5, 6], "out_plot": 2, "outer": 7, "output": [0, 1, 2, 3, 4, 5, 6], "outsid": 4, "p": [5, 6, 7], "p_valu": 1, "packag": 8, "page": 0, "pair": 0, "panel": 7, "paramet": [1, 5, 8], "pathwai": 7, "per": [1, 2, 3, 4, 5, 6], "perform": [4, 6], "permurt": 5, "permut": [4, 5], "piec": 5, "pip": 0, "pleas": [0, 1, 3, 7, 8], "posit": 7, "predefin": 1, "prepar": [2, 3, 5], "preprocess": [0, 8], "previou": 2, "previous": 7, "print": 3, "print_output": 3, "process": [0, 7], "proport": 3, "provid": 8, "pval": 4, "pval_cutoff": [4, 5, 6], "python": [0, 8], "random": [3, 4, 5], "random_se": 3, "randomli": 4, "rang": 1, "ratio": [6, 7], "raw": [1, 2], "reactom": 7, "readm": 8, "recommed": 3, "recommend": [1, 3], "record": 7, "refer": 8, "refin": 3, "region": [0, 1], "rel": [6, 7], "relativeratio": 6, "reorgan": 1, "repres": [0, 7], "repress": 7, "repressor": [6, 7], "requir": [1, 2, 3, 5, 6], "resampl": 4, "resist": 7, "resolut": [0, 1, 2, 3, 4, 5, 6, 7], "resolution_paramet": 3, "result": [0, 3, 4, 5, 7], "right": 7, "ring": 7, "same": [1, 5], "sampl": [4, 5], "score": 3, "screen": [5, 6, 7], "section": [1, 2, 6], "see": 7, "seed": 3, "segment": 7, "select": [3, 4, 5, 6, 7], "set": [2, 5], "sever": 4, "should": [2, 3, 5, 6], "signal": 1, "signific": [1, 5, 6, 7], "significantli": [4, 5], "size": [0, 3, 7], "slm": 3, "smaller": 3, "specif": 3, "standard": 3, "start": [3, 5], "step": [1, 5, 6], "store": [1, 4], "structur": 4, "sub": 7, "subnetwork": 6, "summar": 5, "summari": 3, "super": [1, 3], "super_resolut": 1, "supplementari": 8, "symbol": 7, "system": 0, "t0": 0, "t1": 0, "tabl": 3, "tamoxifen": 7, "tamr": 0, "test": [4, 5, 7], "than": [3, 4], "thei": [2, 6, 7], "them": 7, "therefor": 5, "thi": [1, 3, 6], "threshold": 7, "through": [3, 7], "time": [1, 3, 4], "tool": 7, "top": 7, "topolog": [5, 7], "transform": 3, "tsv": 3, "tval": 4, "two": [2, 3, 5, 7], "type": 1, "type_of_calcul": 2, "under": 7, "untreat": [0, 8], "us": [0, 1, 3, 7], "user": [5, 6], "val": 5, "valid": [3, 4], "valu": [2, 3, 5, 6, 7, 8], "visual": 7, "want": [0, 1, 3, 5, 6], "we": [1, 5], "websit": 6, "well": 6, "when": [4, 5], "where": [0, 1, 3, 7], "whether": [3, 7], "which": 7, "while": 3, "whole": [1, 3], "whole_genom": [0, 1, 3, 5], "window": [0, 2, 3, 7], "within": [3, 4, 6, 7], "written": 0, "ye": 3, "you": [0, 1, 3], "zscore": [1, 2]}, "titles": ["Installation", "Hi-C data preprocessing", "Multi-omics data preprocessing", "Identification of intrachromosomal communities", "Enrichment of genomic feature", "Differential interacting nodes", "Enrichment and network analysis", "Supplementary Files", "DNAICI: Differential Network Analysis in Intra-chromosomal Community Interaction"], "titleterms": {"1": [1, 2, 3, 4, 5, 6], "2": [1, 2, 3, 4, 5, 6], "3": [1, 3, 5], "4": 3, "analysi": [6, 8], "argument": [0, 1, 2, 3, 4, 5, 6], "c": 1, "chromosom": 8, "cluster_community_structur": 3, "cluster_for_h": 3, "cluster_super_network": 3, "commun": [3, 8], "content": [0, 8], "data": [1, 2, 3, 4, 5, 6], "diegs_enrich": 6, "diegs_subnetwork": 6, "differenti": [5, 8], "dnaici": 8, "download": 0, "enrich": [4, 6], "enrichment_heatmap": 4, "enrichment_permut": 4, "estimate_community_s": 3, "estimate_resolut": 1, "featur": 4, "file": 7, "genom": 4, "hi": 1, "identif": 3, "instal": 0, "interact": [5, 8], "intra": 8, "intrachromosom": 3, "multi": 2, "network": [6, 8], "network_centr": 5, "network_comparison": 5, "network_signod": 5, "node": 5, "omic": 2, "packag": 0, "path": [1, 2, 3, 4, 5, 6], "preprocess": [1, 2], "preprocess_hic_homer2b": 1, "preprocess_hic_tag2hom": 1, "preprocess_omics_heatmap": 2, "preprocess_omics_map2h": 2, "requir": 0, "supplementari": 7}})